### ISSUE TITLE
The flat color picker loses the ability to choose a color when it has been detached and reattached from a hidden div in Google Chrome Version 26.0.1410.64 m.

### ISSUE THREAD

**Issue by the1076** [bgrins/spectrum#88]
When using the jQuery functions ".detach()" and ".appendTo()" on a div containing a control that has been replaced with a spectrum color picker (that has the "flat" option set to true), the color selection ability will only select colors in grayscale.
This issue does not occur in IE10 or Firefox.
This issue does not affect spectrum color pickers that do not have the "flat" option set to true.

An example can be seen here. Select "Show" then try to select a red color.
http://jsfiddle.net/NPKBD/1/

There may be an alltogether better way to do this using the new "appendTo" option, but I couldn't quite figure it out yet.
Either way, I figured you might want to know about unexpected functionality.
